packages/extension-browser/src/devtools/views/controls/external-doc.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+import { ProblemDocumentation } from '@hint/utils-types';
+
+import { getMessage } from '../../utils/i18n';
+import ExternalLink from './external-link';
+
+import * as styles from './external-doc.css';
+
+type Props = {
+    doc: ProblemDocumentation[];
+};
+
+const ExternalDoc = ({ doc }: Props) => {
```

Comment:
Nit: Let's make the names plural since this represents a list of documentation URLs.
```suggestion
const ExternalDocs = ({ docs }: Props) => {
```